src/pages/home/home.ts

```diff
@@ -325,21 +341,32 @@ export class HomePage {
     icon,
     lat,
     lng,
+    draggable,
     clickFunction?,
-    clickFunctionParams?
+    clickFunctionParams?,
   ) {
     this.map
       .addMarker({
         title,
         icon,
         animation: "DROP",
+        draggable: draggable,
```

Comment:
**Best Practice**: Since the name of the variable is the same as the name of the object key, you can just put `draggable`, similar to lines 350 and 351. That's called [property shorthand](http://es6-features.org/#PropertyShorthand).